aten/src/ATen/native/native_functions.yaml

```diff
@@ -388,19 +388,19 @@
     SparseCUDA: add_out_sparse_cuda
     MkldnnCPU: mkldnn_add_out
 
-- func: add_relu.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
+- func: _add_relu.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
```

Comment:
should we also make it "python_module: nn"?

Reply:
I wanted to make the smaller change possible here as I don't know exactly where the `nn` version gets exported. This can be done in the follow up PR that will clean up this op?

Reply:
sgtm.